libraries/mne/mne.cpp

```diff
@@ -274,3 +276,23 @@ void MNE::setup_compensators(FiffRawData& raw,
         }
     }
 }
+
+//=============================================================================================================
+
+bool MNE::read_events(QIODevice &p_IODevice,
```

Comment:
Please rename this function to `read_events_from_ascii` and make use of `MatrixXi` instead of a `QList`. The equivalent function which reads from .fif file should be renamed to `read_events_from_fif`. Make use of these two functions in `MNE::read_events(QString t_sEventName, QString t_fileRawName,MatrixXi& events)`